projects/wave-core-new/src/lib/operators/result-type.model.ts

```diff
@@ -24,8 +24,7 @@ class Raster extends ResultType {
 
 class Points extends ResultType {
     getCode(): string {
-        return 'vector';
-        // return 'points'; // TODO: use again when differentiation is possible
+        return 'points';
```

Comment:
maybe you can change it to a readonly public variable here, too. This way we don't create a new string each time.

Reply:
we use this pattern elsewhere too maybe we should have a separate task for this

Reply:
Yeah, however, I would just change it as we go.
It's from legacy code prior to the `readonly` feature of typescript
But if you don't want to do this, you can create a separate task.

Reply:
231dc5ec1652f1707172a5056a1b8b7dac880e04